util/travis/before_install.sh

```diff
@@ -9,24 +9,25 @@ if [ $CC = "clang" ]; then
 	sudo apt-get install clang
 fi
 sudo apt-get install p7zip-full
-if [ $WINDOWS = "no" ]; then
+if [[ $PLATFORM == "Linux" ]]; then
 	sudo apt-get install libirrlicht-dev cmake libbz2-dev libpng12-dev \
-	libjpeg8-dev libxxf86vm-dev libgl1-mesa-dev libsqlite3-dev libhiredis-dev \
-	libogg-dev libvorbis-dev libopenal-dev gettext
+		libjpeg8-dev libxxf86vm-dev libgl1-mesa-dev libsqlite3-dev \
+		libhiredis-dev libogg-dev libvorbis-dev libopenal-dev gettext
 	# Linking to LevelDB is broken, use a custom build
 	wget http://sfan5.pf-control.de/libleveldb-1.18-ubuntu12.04.7z
 	sudo 7z x -o/usr libleveldb-1.18-ubuntu12.04.7z
-else
-	if [ $WINDOWS = "32" ]; then
-		wget http://sfan5.pf-control.de/mingw_w64_i686_ubuntu12.04_4.9.1.7z -O mingw.7z
-		sed -e "s|%PREFIX%|i686-w64-mingw32|" \
-			-e "s|%ROOTPATH%|/usr/i686-w64-mingw32|" \
-			< util/travis/toolchain_mingw.cmake.in > util/buildbot/toolchain_mingw.cmake
-	elif [ $WINDOWS = "64" ]; then
-		wget http://sfan5.pf-control.de/mingw_w64_x86_64_ubuntu12.04_4.9.1.7z -O mingw.7z
-		sed -e "s|%PREFIX%|x86_64-w64-mingw32|" \
-			-e "s|%ROOTPATH%|/usr/x86_64-w64-mingw32|" \
-			< util/travis/toolchain_mingw.cmake.in > util/buildbot/toolchain_mingw64.cmake
-	fi
+elif [[ $PLATFORM == "Win32" ]]; then
+	wget http://sfan5.pf-control.de/mingw_w64_i686_ubuntu12.04_4.9.1.7z -O mingw.7z
+	sed -e "s|%PREFIX%|i686-w64-mingw32|" \
+		-e "s|%ROOTPATH%|/usr/i686-w64-mingw32|" \
+		< util/travis/toolchain_mingw.cmake.in > util/buildbot/toolchain_mingw.cmake
+elif [[ $PLATFORM == "Win64" ]]; then
+	wget http://sfan5.pf-control.de/mingw_w64_x86_64_ubuntu12.04_4.9.1.7z -O mingw.7z
+	sed -e "s|%PREFIX%|x86_64-w64-mingw32|" \
+		-e "s|%ROOTPATH%|/usr/x86_64-w64-mingw32|" \
+		< util/travis/toolchain_mingw.cmake.in > util/buildbot/toolchain_mingw64.cmake
+fi
+if [[ $PLATFORM == Win* ]]; then
 	sudo 7z x -y -o/usr mingw.7z
 fi
```

Comment:
This is 1 line longer than just duplicating the command.
Also this "disconnects" it from the rest of the windows-specific code making it less obvious that it has something to do w/ it.